### PR TITLE
[OP#49295]Change NCMultiselect to NcSelect

### PR DESCRIPTION
**Pull request by SagarGi**
### Description
This PR replaces the deprecated version of `NcMultiSelect` to `NcSelect` nextcloud-vue component.

### Related Work Packages
https://community.openproject.org/projects/nextcloud-integration/work_packages/49295/activity?query_id=3787

### Before:

Screenshot from 2023-07-31 14-38-57

### After:

Screenshot from 2023-07-31 14-38-14

### CODE REVIEW

File: tests/jest/components/tab/SearchInput.spec.js [hunk 1]
```diff
@@ -176,15 +176,22 @@ describe('SearchInput.vue', () => {
...
 			})
 			it('should display correct options list of search results', async () => {
+				jest.spyOn(axios, 'get')
+					.mockImplementationOnce(() => Promise.resolve({
+						status: 200,
+						data: [],
+					}))
+				wrapper = mountSearchInput({ id: 1234, name: 'file.txt' })
+				const inputField = wrapper.find(inputSelector)
+				await inputField.setValue(' ')
```
Comment (SagarGi) [inline tests/jest/components/tab/SearchInput.spec.js:190]: for the search result to set in the `dropdown` it needs to be opened. Setting the value with empty value we can get the dropdown open which gives the workpackages in the search result.

Comment (SagarGi) [inline tests/jest/components/tab/SearchInput.spec.js:190]: For the search data to be visible in the dropdown. The dropdown needs to be opened. So this helps to open it. So after that we can check the content of the dropdown based on the search result.

**Review by SwikritiT — APPROVED** [review]
LGTM

File: tests/jest/components/tab/SearchInput.spec.js [hunk 2]
```diff
@@ -176,15 +176,22 @@ describe('SearchInput.vue', () => {
...
 				await wrapper.setData({
-					fileInfo: { id: 1234 },
 					searchResults: workPackagesSearchResponse,
 				})
-				const multiSelectContent = wrapper.find(multiSelectContentSelector)
+				const multiSelectContent = wrapper.find(workpackagesListSelector)
```
Comment (SwikritiT) [inline tests/jest/components/tab/SearchInput.spec.js:194]: It's not multiseletc anymore so can we change the name to something more appropriate 
```suggestion
				const ncSelectContent = wrapper.find(workpackagesListSelector)
```

Comment (SwikritiT) [inline tests/jest/components/tab/SearchInput.spec.js:194]: It's not multiseletc anymore so can we change the name to something more appropriate 
```suggestion
				const ncSelectContent = wrapper.find(workpackagesListSelector)
```

File: tests/jest/components/tab/__snapshots__/SearchInput.spec.js.snap [hunk 3]
```diff
@@ -36,15 +36,7 @@ Object {
...
-exports[`SearchInput.vue work packages multiselect search list should not be displayed if the search results is empty 1`] = `
-<ul role="listbox" id="listbox-null" class="multiselect__content" style="display: block;">
-  <!---->
-  <li style="display: none;"><span class="multiselect__option"><span>No results</span></span></li>
-  <li><span class="multiselect__option">
-			Start typing to search
-		</span></li>
-</ul>
-`;
+exports[`SearchInput.vue work packages multiselect search list should not be displayed if the search results is empty 1`] = `<ul id="vs10__listbox" role="listbox" style="display: none; visibility: hidden;"></ul>`;
```
Comment (SwikritiT) [inline tests/jest/components/tab/__snapshots__/SearchInput.spec.js.snap:39]: We need to find the occurance of the word multiselect and replace/remove it. This probably need the chane in the title of the unit test
```suggestion
exports[`SearchInput.vue work packages  search list should not be displayed if the search results is empty 1`] = `<ul id="vs10__listbox" role="listbox" style="display: none; visibility: hidden;"></ul>`;
```

**Review by SwikritiT — CHANGES_REQUESTED** [review]

**Review by SwikritiT — APPROVED** [review]
LGTM

**Pull request merged**